app/ui/webpack.config.hot.js

```diff
@@ -0,0 +1,23 @@
+var config = require('./webpack.config.base');
```

Comment:
What I meant, @jameshopkins, is the value is mutable (so can't/shouldn't be a `const`) , but it totally can be a `let`, although I can't help feeling this level of focus on a config file is a bit OCD 😄